src/test/java/api/support/fakes/FakeOkapi.java

```diff
@@ -463,6 +477,16 @@ private void registerCirculationRulesStorage(Router router) {
     });
   }
 
+  private void registerOpeningHours(Router router) {
+    router.get("/calendar/periods")
+      .handler(routingContext -> {
+        routingContext.response()
+          .setStatusCode(200)
+          .putHeader("content-type", "application/json")
+          .end(OpeningPeriodsExamples.oneDayPeriod().create().toString());
+      });
+  }
```

Comment:
I think it could make sense to extract methods for all of the calendar related setup and move them to a separate class, in order to make it clear these use a different style of integration and so the fakes are different

Reply:
Moved calendar related code to FakeCalendarOkapi